frontend/src/Components/Routes/ProtectedRoute.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Navigate } from "react-router-dom";
+
+const ProtectedRoute = ({ accessToken, children }) => {
+	if (!accessToken) {
+		return <Navigate to={"/login"} />;
+	}
+	return children;
+};
+
+export default ProtectedRoute;
```

Comment:
This is good. One more comment you can here define Route also:
`  if (! accessToken) {
		return <Navigate to={"/login"} />;
}
  return (
    <Route {...routeProps}>
        {children}
    </Route>
  );
};`